Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 RAILS_VERSION = '~> 4.1.8'
 
-ruby ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
+send :ruby, ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
```

Comment:
would `ruby(ENV['GEMFILE_RUBY_VERSION']) if ENV['GEMFILE_RUBY_VERSION']` 
(adding parenthesis) fix the problem?